charts/rstudio-pm/README.md.gotmpl

```diff
@@ -7,6 +7,18 @@
 
 {{ template "rstudio.install" . }}
 
+## Required Configuration
+
+This chart requires the following in order to function:
+
+* A license key, license file, or address of a running license server. See the `license` configuration below.
+* A Kubernetes [PersistentVolume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) that contains the data directory for RSPM.
+  * If `sharedStorage.create` is set, a PVC that relies on the default storage class will be used. Most Kubernetes environments do not have a default
+    storage class that you can use with `ReadWriteMany` access mode out-of-the-box. In this case, we recommend you disable `sharedStorage.create` and 
+    create your own `PersistentVolume` and `PersistentVolumeClaim`, then mount them into the container by specifying the `pod.volumes` and `pod.volumeMounts` parameters.
+  * If you cannot use a `PersistentVolume` to properly mount your data directory, you'll need to manually mount your NFS server in the container during startup
```

Comment:
@colearendt have we experimented with configuring RSPM to run with S3 for its shared data store from a Helm chart install?

Reply:
"manually mount your NFS server to the container during startup" can be done with an NFS volume (i.e. using `pod.volumes` and `pod.volumeMounts` directly without a PV/PVC)

https://kubernetes.io/docs/concepts/storage/volumes/#volume-types

@jonyoder Yes we have! In fact, this is what we do for https://colorado.rstudio.com/rspm/ - so that is definitely a case we should mention for RSPM